### PR TITLE
Add @Timestamped annotation

### PR DESCRIPTION
**Pull request by zikani03**
Added @Timestamped Statement Customizer to add current datetime as
parameter for use in queries, tests included :)

### CODE REVIEW

File: sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestTimestamped.java [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        p.setId(1);
+        personDAO.insert(p);
+
+        Person found = personDAO.get(1);
+
+        assertThat(found.getCreated()).isNotNull();
+        assertThat(found.getModified()).isNotNull();
+
+        // This is one way we can get the binding information of the executed query
+        h2DatabaseRule.getJdbi().setTimingCollector((l, statementContext) -> {
```
Comment (qualidafial) [inline sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestTimestamped.java]: This timing collector needs to be configured before the `personDAO.insert()` call for the assertion to actually be tested. (applies to all 3 test methods)

**Comment by qualidafial** [comment]
@zikani03 Try pulling from upstream `jdbi3` branch and push your merge commit--that should fix the build.

**Comment by qualidafial** [comment]
Thanks for contributing!

**Comment by stevenschlansker** [comment]
Very nice!

**Pull request merged**